doc/advanced_configuration.rst

```diff
@@ -13,51 +13,70 @@ Within your Sphinx ``conf.py`` file you need to add a configuration dictionary:
 .. code-block:: python
 
     sphinxgallery_conf = {
-        'root_dir'          : '../examples',              # path to your examples scripts
-        'examples_gallery'  : 'auto_examples'}            # path where to save gallery generated examples
+        'examples_dir' : '../examples',              # path to your examples scripts
+        'gallery_dir'  : 'auto_examples'}            # path where to save gallery generated examples
 
 
 Directory paths are relative to your ``conf.py`` location.
 
+
 Linking to external documentations
 ==================================
 
-If you want to hyperlink commands in your example scripts you can. Again within
-your Sphinx ``conf.py`` file you need to add a configuration dictionary:
+Sphinx-Gallery enables you to hyperlink commands in your example scripts to the
+matching location in their online documentation . Have a look at this in action
+in our example :ref:`example_plot_gallery_version.py`.
```

Comment:
This sounds quite vague to me and it actually took me a while to figure out what that actually meant, i.e. that some functions in the example html were clickable to access the online documentation of packages used in the example, e.g. numpy, matplotlib, etc .... Maybe it would help to add a few words why this is useful or mention an explicit line to click on in the example.

Loosely related it doesn't help that clickable functions are not visible in the example. I agree that for readability underline is not necessarily the best option. Maybe some CSS tricks on hover would be good enough.